lms/static/sass/discussion/_layouts.scss

```diff
@@ -0,0 +1,40 @@
+// Layouts for discussion pages
+
+section.user-profile {
+    background-color: $sidebar-color;
+
+    .user-profile {
+        padding: 32px 36px;
```

Comment:
These values seem fishy. Is there any way to make them relative based on our `$baseline` value?